<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget  {
	background-color: black;
 }</string>
   </property>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>344</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #ab6f00;
	font: 48px;
	color: #f5f5f5;
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QFrame" name="ERR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_4">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>Errors</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ERRtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="OilP">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_8">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>Oil Pressure</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="OilPtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>300.5 kpa</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="CT">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_3">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>Coolant Temp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="CTtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>89.5 Â°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="MAP">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_7">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>MAP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="MAPtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>99.9 kpa</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="RPM">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_10">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>RPM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="RPMtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>8000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="OilT">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_9">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>Oil Temp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="OilTtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>70.5 Â°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="BV">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>Voltage</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="BVtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>12.00 V</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="AFR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>AFR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AFRtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>13.85</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="IA">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>344</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_5">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>Ignition Angle</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="IAtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>10.5 btd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="TPS">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>344</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_11">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>TPS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="TPStxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>100.0 %</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="IAT">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>344</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
 border: 4px solid #383838;
 background: #262626;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_6">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>333</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 32px;
	color: #faae20;
}</string>
     </property>
     <property name="text">
      <string>IAT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="IATtxt">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>54</y>
       <width>333</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent; border: 0;
	font: 48px;
	color: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string>20.5 Â°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
